### ISSUE TITLE
Product Option Saving Issue

### ISSUE THREAD

**Issue by indclassictrophy** [opencart/opencart#2748]
I have all of my options set up in the options area but when I go to add an option group to an actual product it won't saving past 96 options. Arvixe has stated that my settings are correct and I shouldnt have any issues. I am using OpenCart 2.0.1.1. This issue is stopping me from creating products.

**Comment by indclassictrophy** [comment]
Daniel, I havent used this site before - does closed mean someone is working on it?

**Comment by milimar** [comment]
Unfortunately,  closing an issue without an explanation doesn't mean that someone is working on it. It means that Daniel considers that the issue is not a bug.   
You can try to get help at the forum of OC http://forum.opencart.com . 

Greetings

**Comment by indclassictrophy** [comment]
hmmm I increased the max_input_vars up even more to see if that would work and it doesnt. All my option groups and their options are saved in the option area of open cart but when I go to add them to a product it wont save past 96 options. Any ideas? I've talked and asked everything I could think of now.

Thanks!Zac Weidler
Marketing Director
 Classic Trophy Co., Inc.
Celebrating 50 Years 1965-2015  www.indclassictrophy.comwww.indclassicshirts.com  zac@indclassictrophy.com 260.483.1161 

----------------------------------------
